src/pages/index.js

```diff
@@ -0,0 +1,249 @@
+
+const postModal = document.querySelector("#post-modal");
+const postForm = document.forms["post-form"];
+const postSubmitButton = postModal.querySelector(".modal__submit-button");
+const postModalLink = postModal.querySelector("#modal-link");
+const postModalCaption = postModal.querySelector("#modal-caption");
+
+const cardTemplate = document.querySelector("#card-template");
+const cardsList = document.querySelector(".cards__list");
+
+const deleteModal = document.querySelector("#delete-modal");
+const deleteForm = deleteModal.querySelector("#delete-form");
+const cancelDeleteButton = deleteModal.querySelector(".modal__cancel-button");
+
+
+const photoModal = document.querySelector("#photo-modal");
+const photoModalImage = photoModal.querySelector(".modal__image");
+const photoModalCaption = photoModal.querySelector(".modal__caption");
+
+
+
+
+let selectedCard;
+let selectedCardId;
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "dbd91921-c78f-4c03-af14-561b6abbc01c",
+    "Content-Type": "application/json"
+  }
+});
+
+
+api.getAppInfo()
+.then(([cards, userInfo]) => {
+  cards.forEach(function (initialCards) {
+  renderCard(initialCards, "append");
+});
+  profileAvatar.src = userInfo.avatar;
+  profileName.textContent = userInfo.name;
+  profileDescription.textContent = userInfo.about;
+})
+.catch(console.error);
+
+function handleLike(evt, cardId) {
+  const isLiked = evt.target.classList.contains("card__like-button_liked");
+  api.changeLikeStatus(cardId, isLiked)
+  .then((res) => {
+    evt.target.classList.toggle("card__like-button_liked");
+    })
+  .catch(console.error);
+}
+
+function handleDeleteCard(cardElement, cardId) {
+  selectedCard = cardElement;
+  selectedCardId = cardId;
+  openModal(deleteModal);
+}
+
+function handleDeleteSubmit(evt) {
+    evt.preventDefault();
+
+    const submitButton = evt.submitter;
+    setButtonText(submitButton, true, "Delete", "Deleting...");
+
+    api
+    .deleteCard(selectedCardId)
+    .then(() => {
+      selectedCard.remove();
+      closeModal(deleteModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+    setButtonText(submitButton, false, "Delete", "Deleting...");
+    })
+};
+
+cancelDeleteButton.addEventListener("click", () => closeModal(deleteModal));
+deleteForm.addEventListener("submit", handleDeleteSubmit);
+
+function getCardElement(data) {
+  const cardElement = cardTemplate.content.cloneNode(true).querySelector('.card');
+  const cardNameElement = cardElement.querySelector(".card__title");
+  const cardImageElement = cardElement.querySelector(".card__image");
+
+  const cardLikeButton = cardElement.querySelector(".card__like-button");
+  const cardDeleteButton = cardElement.querySelector(".card__delete-button");
+
+  cardNameElement.textContent = data.name;
+  cardImageElement.src = data.link;
+  cardImageElement.alt = data.name;
+
```

Comment:
* When I reload the page, all **red** hearts become **transparent** again.


Please, check `isLiked` here